go/chat/ephemeral_purger.go

```diff
@@ -256,7 +256,7 @@ func (b *BackgroundEphemeralPurger) loop() {
 			b.queueLock.Unlock()
 		case <-b.outboxTicker.C:
 			// Check the outbox for stuck ephemeral messages that need purging
-			storage.NewOutbox(b.G(), b.uid).EphemeralPurge(context.Background())
+			storage.NewOutbox(b.G(), b.uid).OutboxPurge(context.Background())
```

Comment:
I'm not too excited about this being a general thing invoked from the exploding purger. Is there any other way that isn't too much work so that we can decouple this error purger from this loop?

Reply:
we can move it back into the service i don't think it would be that much extra